spec/RestQuery.spec.js

```diff
@@ -521,7 +521,6 @@ describe('RestQuery.each', () => {
       'createdAt',
       'initialToRemove',
       'objectId',
-      'updatedAt',
```

Comment:
Could you explain why this removal?

Reply:
When we added `skipKeys` previously, we assumed that updatedAt would always be returned. However, on create, Parse Server only returns objectId and createdAt. This change aligns the save response for with and without cloud triggers. 

I don’t think this is a breaking change, it’s just that the PR that introduced skipKeys also introduced returning updatedAt on create, which is outside the usual behaviour of Parse Server. 

There was a lack of tests to ensure that the save response’s keys are the same regardless of if triggers are defined or not. This resulted in many keys being returned that were not previously. This has been addressed in this PR

Reply:
> I don’t think this is a breaking change, it’s just that the PR that introduced skipKeys also introduced returning updatedAt on create, which is outside the usual behaviour of Parse Server.

Got it, I also don't think it's a breaking change because of what you said above.